hasweb/templates/index.html

```diff
@@ -47,6 +47,12 @@
     {%- for etitle, edate, eurl, edesc in [
         ('Rootconf 2016', 'Thu, Apr 14 – Fri, Apr 15, Bangalore', 'https://rootconf.talkfunnel.com/2016/',
           'On learning from failure in DevOps'),
+        ('​Introduction to Data Science in Python', 'March 12 - 13, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-python-1/',
```

Comment:
Think it's better for this to be listed before Rootconf, adhering the chronological order.